### ISSUE TITLE
Check and verify full script with different parameters

### ISSUE THREAD

**Issue by Sebi94nbg** [TS3Tools/TS3UpdateScript#20]
Make sure, that the script is working with the new changes, before publish.

**Comment by Sebi94nbg** [comment]
Check unzipping of different archives once again:

```
if [ -f teamspeak3-server.tar.bz2 ]; then
    bzip2 teamspeak3-server.tar.bz2
    tar xf teamspeak3-server.tar.bz2
fi

if [ -f teamspeak3-server.tar.gz ]; then
    tar xf teamspeak3-server.tar.gz
fi
```
https://github.com/TS3Tools/TS3UpdateScript/blob/master/TS3UpdateScript#L1554-L1586

**Comment by Sebi94nbg** [comment]
Make sure, that there is no non-unix format like **^M** by running this command:

`dos2unix TS3UpdateScript`

**Comment by Sebi94nbg** [comment]
Verifing these solved issues:
#16 
#17 
#18

**Comment by Sebi94nbg** [comment]
Following test was successful:

> ./TS3UpdateScript --check --delete-old-logs --inform-online-clients

```
Poking clients on following instance: /home/teamspeak [ OK ]

Temporary server passwords were backuped successful.
TeamSpeak 3 server instance stopped for update process.
Deleting old log files.
Backup created successful.
TeamSpeak 3 server instance was updated successful.
TeamSpeak 3 server instance was started successful.
Temporary server passwords were restored successful.
Waiting 15 seconds to execute TeamSpeak 3 server and TSDNS health check...
TeamSpeak 3 server instance is still running.
Backup was deleted successful.
```

**Comment by Sebi94nbg** [comment]
https://github.com/TS3Tools/TS3UpdateScript/issues/20#issuecomment-183353869 was checked successful.

**Comment by Sebi94nbg** [comment]
Following test was **successful** except restoring temporary passwords:

> ./TS3UpdateScript --check --delete-old-logs

Installed software packages: telnet, expect

```
Temporary server passwords were backuped successful.
TeamSpeak 3 server instance stopped for update process.
Deleting old log files.
Backup created successful.
TeamSpeak 3 server instance was updated successful.
TeamSpeak 3 server instance was started successful.
Could not restore temporary server passwords.
Waiting 15 seconds to execute TeamSpeak 3 server and TSDNS health check...
TeamSpeak 3 server instance is still running.
Backup was deleted successful.
```

**Comment by Sebi94nbg** [comment]
Fixed #21
Fixed "Could not restore temporary server passwords." (https://github.com/TS3Tools/TS3UpdateScript/issues/20#issuecomment-183456465).

Following test was **successful**:

> ./TS3UpdateScript --check --delete-old-logs

Installed software packages: telnet, expect

```
Temporary server passwords were backuped successful.
TeamSpeak 3 server instance stopped for update process.
Deleting old log files.
Backup created successful.
TeamSpeak 3 server instance was updated successful.
TeamSpeak 3 server instance was started successful.
Temporary server passwords were restored successful.
Waiting 15 seconds to execute TeamSpeak 3 server and TSDNS health check...
TeamSpeak 3 server instance is still running.
Backup was deleted successful.
```

**Comment by Sebi94nbg** [comment]
Following test was **successful**:

> ./TS3UpdateScript --check --delete-old-logs --inform-online-clients --keep-backups --waiting-time-in-minutes 2

```
Poking clients on following instance: /home/teamspeak [ OK ]

2 minutes to wait...
Temporary server passwords were backuped successful.
TeamSpeak 3 server instance stopped for update process.
Deleting old log files.
Backup created successful.
TeamSpeak 3 server instance was updated successful.
TeamSpeak 3 server instance was started successful.
Temporary server passwords were restored successful.
Waiting 15 seconds to execute TeamSpeak 3 server and TSDNS health check...
TeamSpeak 3 server instance is still running.
The backup for this instance wasn't deleted. Backup can be found here: /tmp/TS3Tools/TS3UpdateScript/5bc202c88e6448e17d805cc22d2c5c1c/
```

**Comment by Sebi94nbg** [comment]
Following test was **successful**:

> ./TS3UpdateScript --check --delete-old-logs --inform-online-clients --keep-backups --waiting-time-in-minutes 2

```
Poking clients on following instance: /home/teamspeak [ OK ]

2 minutes to wait...
TSDNS stopped for update process.
Temporary server passwords were backuped successful.
TeamSpeak 3 server instance stopped for update process.
Deleting old log files.
Backup created successful.
TeamSpeak 3 server instance was updated successful.
TSDNS was started successful.
Scanned tsdns_settings.ini, number of entries: 1 normal and 0 wildcards
TeamSpeak 3 server instance was started successful.
Waiting 15 seconds to execute TeamSpeak 3 server and TSDNS health check...
TeamSpeak 3 server instance is still running.
TSDNS is still running.
The backup for this instance wasn't deleted. Backup can be found here: /tmp/TS3Tools/TS3UpdateScript/5bc202c88e6448e17d805cc22d2c5c1c/
```

**Comment by Sebi94nbg** [comment]
All tests were successful.

A test counts as "unsuccessful" if the update would cause a crashed TeamSpeak 3 server.

Issues which appeared while testing but didn't broke the TS3 installation were fixed.

**Comment by Sebi94nbg** [comment]
Released this under v4.3.1: https://github.com/TS3Tools/TS3UpdateScript/releases/tag/4.3.1
